### PR TITLE
PMA-85 translating all

### PR DESCRIPTION
**Pull request by PavelZabalotny**
Translated: boards, boards-header, board, column, task components and boards, columns, tasks, confirmation modals.

### CODE REVIEW

File: src/app/boards/components/column/column.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<mat-card class="column-card" cdkDrag [cdkDragData]="column">
+<mat-card class="column" cdkDrag [cdkDragData]="column">
   <div class="buttons-group">
-    <div class="dragHandle" cdkDragHandle>
+    <div class="example-handle" cdkDragHandle>
```
Comment (Manofsky) [inline src/app/boards/components/column/column.component.html]: must be a `dragHandle` selector

File: src/app/boards/components/column/column.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<mat-card class="column-card" cdkDrag [cdkDragData]="column">
+<mat-card class="column" cdkDrag [cdkDragData]="column">
```
Comment (Manofsky) [inline src/app/boards/components/column/column.component.html]: must be a `column-card` selector

**Comment by Manofsky** [comment]
Scrolling in columns does not work

**Review by Manofsky — CHANGES_REQUESTED** [review]
Scrolling in columns & in boards does not work.
It seems to me it is necessary to compare the versions of files in which there were conflicts and fix bugs.

**Review by Manofsky — APPROVED** [review]

File: src/app/boards/components/column/column.component.html [hunk 3]
```diff
@@ -58,7 +58,7 @@
           mat-icon-button
           color="primary"
           aria-label="done button"
-          matTooltip="Submit the Title"
+          matTooltip="{{ 'icons.submit' | translate }}"
           matTooltipPosition="above"
```
Comment (BlueOwll) [inline src/app/boards/components/column/column.component.html]: i suppose "below" would be better in order to not mix tooltips
or "right" for both of buttons

**Review by BlueOwll — APPROVED** [review]

**Pull request merged**